java/src/main/java/ai/rapids/cudf/WindowOptions.java

```diff
@@ -33,8 +50,10 @@ enum FrameType {ROWS, RANGE}
   private final int orderByColumnIndex;
   private final boolean orderByOrderAscending;
   private final FrameType frameType;
-  private final boolean isUnboundedPreceding;
-  private final boolean isUnboundedFollowing;
+  private final RangeExtentType precedingBoundsExtent;
+  private final RangeExtentType followingBoundsExtent;
+//  private RangeExtentType precedingBoundsExtent = RangeExtentType.BOUNDED;
+//  private RangeExtentType followingBoundsExtent = RangeExtentType.BOUNDED;
```

Comment:
Need cleanup?

Reply:
Thank you. Yes, I should've removed that.

Reply:
Removed now.